config.example.js

```diff
@@ -31,12 +31,12 @@ module.exports = {
   symbol: 'SYM',
   // Project description
   description: 'An awesome NFT series',
-  // Path to image to use as the main image for the collection
+  // Path to image to use as the main image for the collection. gif compatible
   // (at least 500 x 500 pixels)
   image: 'images/1.png',
```

Comment:
We should probably change this default so that it's an IPFS URI.

Reply:
does that conflict with the rust contract https://github.com/public-awesome/stargaze-contracts/blob/main/contracts/sg721/src/contract.rs#L63

Reply:
This doesn't need to be an IPFS URL but it can. Read the 2nd paragraph here: https://github.com/public-awesome/stargaze-contracts/pull/96#issue-1155169657. We also want people to be able to easily change it for reveal-later projects, or have teaser placeholders in the future.